testsuite/ui/widgets/__init__.py

```diff
@@ -441,3 +458,29 @@ def text(self):
         """Get text of metric"""
         self.wait_displayed()
         return self.browser.element(self.locator).text
+
+
+class ThreescaleButtonGroup(GenericLocatorWidget):
+    """Specific Button Group of 3scale pages"""
+
+    SELECTED = "./a[1]"
+    DROPDOWN_ITEMS = "./ul/li/a"
+    DROPDOWN_TOGGLE = "./a[2]"
+
+    def __init__(self, parent=None,
+                 locator=".//*[@id='cms-new-content-button']",
```

Comment:
I would suggest passing the locator to it, this default locator is specific for only one button group in 3scale in the developer portal section